src/@types/range_check/index.d.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ *
+ */
+
+declare module "range_check";
```

Comment:
can we have more comprehensive types for `range_check` too? (like you did for `passport-auth-token`)